README.md

```diff
@@ -8,4 +8,5 @@ This project is part of ASP.NET 5. You can find samples, documentation and getti
 ## Docs and samples
```

Comment:
If we're going to add links to this section we should probably name it more appropriately because neither of these links are official (even though Louis works here :smile: ). Can we rename this heading to something like `## Blog posts` so that it doesn't sound too official?